### PR TITLE
Add some resources and modify intro text

### PR DESCRIPTION
**Pull request by shalithasuranga**
Hey.. @achou11 Please check and squash then merge (I've done several commits to update stuff) ;). Also, let me know if I need to update something in this PR :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
-# Awesome Neutralino
+# Awesome Neutralinojs [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
```
Comment (achou11) [inline README.md:1]: Should I rename the repo to match? Didn't realize that the `js` was part of the official name 😅

File: README.md [hunk 2]
```diff
@@ -9,35 +17,52 @@ Collection of awesome resources for [Neutralino](https://neutralino.js.org/)
...
 
 ## Apps
 
-- [LiteMan](https://github.com/codezri/liteman) - _A simple file viewer app written in Neutralinojs._
-- [LitePy](https://github.com/codezri/litepy) - _World's simplest Python editor built with Neutralinojs_
+- [LiteMan](https://github.com/codezri/liteman) - A simple file viewer app written in Neutralinojs
+- [LitePy](https://github.com/codezri/litepy) - World's simplest Python editor built with Neutralinojs
+- [Drums](https://github.com/m00ke5h/Drums) - A Neutralino app for playing drums
+- [SimpleTime](https://github.com/navidmafi/SimpleTime) - A cross-platform timer and stopwatch app with modern design in mind
+- [MyoroEditor](https://github.com/antonkoetzler/MyoroEditor) - A simple text editor using the lightweight API NeutralinoJS
```
Comment (achou11) [inline README.md]: Thoughts on making this section (and maybe others too) alphabetically sorted?

File: media/.gitkeep [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (achou11) [inline media/.gitkeep]: is this file necessary anymore since you have the logo now?

**Review by achou11 — APPROVED** [review]
Awesome! Thanks for the changes. Added a few non-urgent comments that might be worth looking at before merging :)

File: README.md (hunk 1 above)
Comment (shalithasuranga) [reply README.md:1]: Hey.. mm I think "Neutralinojs" looks nice. but you can use repo name as you wish :D .. I suggested "Neutralinojs" since we use that name everywhere in documentation, etc

File: README.md (hunk 2 above)
Comment (shalithasuranga) [reply README.md]: Great idea, I will update the list

File: media/.gitkeep (hunk 3 above)
Comment (shalithasuranga) [reply media/.gitkeep]: ah looks like I created it initially for uploading :D .. will remove

**Comment by shalithasuranga** [comment]
Hey.. @achou11  Thanks for checking. I've updated some stuff. could you check :)

File: README.md (hunk 1 above)
Comment (achou11) [reply README.md:1]: gotcha. I'll probably keep the repo name as is in that case. seems analogous to the fact that Electron can also be referred to as Electron.js 😄
